ttnn/cpp/pybind11/__init__.cpp

```diff
@@ -43,5 +43,6 @@ PYBIND11_MODULE(_ttnn, module) {
     module.attr("CONFIG") = &ttnn::CONFIG;
 
     module.def("get_operation_id", []() { return ttnn::OPERATION_ID; }, "Get operation id");
+    module.def("set_operation_id", [](int operation_id) { ttnn::OPERATION_ID = operation_id; }, "Set operation id");
```

Comment:
Is there any point in having it on c++ side since real data source is DB?

Reply:
I believe the intent here is to provide the operation_id to the device level csv so that the exact ttnn call can be associated to the device level code.   In many cases the same operation_id would be associated to muiltiple device operations.